### DISCUSSION TITLE
Apply filter on table column

### DISCUSSION THREAD

**Discussion by dickensas** [NG-ZORRO/ng-zorro-antd#7831] [Q&A] [answered]
When I see the demo https://ng.ant.design/components/table/en#components-table-demo-sort-filter

Under Name column filter "Jim" is already applied, 

But how this can be achieved in code, please help me

![image](https://user-images.githubusercontent.com/4033331/218760139-8471f916-0df1-4267-9a15-dd0619ae8434.png)

**Comment by Edgar-Chmylko-WE** [comment, marked as answer]
Hello,
in typescript file you need to specify which value is default. I will add some screenshots.

![image](https://user-images.githubusercontent.com/117636642/220035923-e807bca9-5649-4bee-9265-975f7ba9e753.png)
![image](https://user-images.githubusercontent.com/117636642/220036003-2f008def-ed34-43fb-9803-3d58b6ab2784.png)

I hope this will help you :)

**Comment by dickensas** [comment]
thanks you this is working
